lnrpc/routerrpc/router.yaml

```diff
@@ -48,3 +48,10 @@ http:
     - selector: routerrpc.Router.UpdateChanStatus
       post: "/v2/router/updatechanstatus"
       body: "*"
+    - selector: routerrpc.Router.XAddLocalChanAliases
+      post: "/v2/router/addaliases"
```

Comment:
We should add the experimental `/x/` prefix here as well, as we do in the `XImportMissionControl`. So `/v2/router/x/addaliases` and `/v2/router/x/deletealiases`.